adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -50,12 +90,7 @@ def htonl(x: int) -> int:
 
     :return int: 32-bit positive integer in network byte order.
     """
-    return (
-        ((x) << 24 & 0xFF000000)
-        | ((x) << 8 & 0x00FF0000)
-        | ((x) >> 8 & 0x0000FF00)
-        | ((x) >> 24 & 0x000000FF)
-    )
+    return int.from_bytes(x.to_bytes(4, "little"), "big")
```

Comment:
Should this be using `sys.byteorder` (or something similar) instead of assuming that it's always running on a little-endian host? I see the previous implementation had that same assumption, so maybe this is a separate Issue, but I wanted to ask.

https://docs.circuitpython.org/en/latest/docs/library/sys.html#sys.byteorder

Reply:
Hi, excellent point. I inherited the assumption, just found a more efficient implementation. I'll check the endidness and only convert if necessary.
Thanks!